cli/flags.go

```diff
@@ -228,6 +228,13 @@ func initFlags(ctx *server.Context) {
 		f := cmd.Flags()
 		f.Int64Var(&maxResults, "max-results", defaultMaxResults, flagUsage["max-results"])
 	}
+
+	// range flag for delete.
+	for _, cmd := range []*cobra.Command{delCmd} {
```

Comment:
iterating over just one command is probably not necessary.
```go
{
  f := delCmd.Flags()
  f.BoolVar(&delRange, "range", false, flagUsage["range"])
}
```

You also need to add something to `flagUsage` under the key "range" though I think you'll want a more descriptive name than "range".